WalletWasabi/Crypto/Randomness/MockRandom.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Crypto.Randomness
+{
+	public class MockRandom : IWasabiRandom
```

Comment:
If this class is to be used in unit tests, then maybe it should be simply abstract class without any methods to let user define the methods as needed. However, then the utility of the class is very small and maybe user can write it herself as needed.

Reply:
It's up for the user how this class evolves. I'd prefer to not police anything for now.